mobius-core/src/main/java/com/spotify/mobius/EffectRouterBuilderImpl.java

```diff
@@ -31,6 +31,23 @@
 
   private final List<Connectable<F, E>> connectables;
   private final List<Class<?>> registeredClasses;
+  private Function<Connectable<? extends F, E>, Consumer<Throwable>> errorHandler =
+      new Function<Connectable<? extends F, E>, Consumer<Throwable>>() {
+        @Nonnull
+        @Override
+        public Consumer<Throwable> apply(Connectable<? extends F, E> value) {
```

Comment:
In the Rx version, the transformer's class name was used to through a `ConnectionException`. Why would this be different? Also, I'm not entirely sold on the signature of this function. I don't know if a `Connectable` instance will be useful in identifying the offending connectable, especially if I use the simpler builder tools such that use action, consumer and function. I wasn't super convinced in the Rx implementation and I'm still not seeing this here.

Reply:
Yeah, I picked that signature for consistency with the Rx version. The idea being that it might be useful to get some information from the handler that threw an exception if you want to try to handle it, but I sort of think that's overthinking the problem and just supplying a consumer is easier and good enough.

Reply:
Regarding ConnectionException or not, I picked this way to avoid having to wrap the exception. But I guess it's a bit easier to always wrap, will do that.